ruma-events/Cargo.toml

```diff
@@ -27,7 +27,13 @@ maplit = "1.0.2"
 matches = "0.1.8"
 ruma-identifiers = { version = "0.16.2", path = "../ruma-identifiers", features = ["rand"] }
 trybuild = "1.0.30"
-criterion = "0.3.2"
+
+[dependencies.criterion]
```

Comment:
Can you make this an inline table like `ruma-identifiers`, so every dependency is just one line?